src/components/SidePane/Help/index.tsx

```diff
@@ -1,25 +1,84 @@
-import React from 'react';
+import React, {useEffect} from 'react';
 // eslint-disable-next-line no-restricted-imports
-import {Animated} from 'react-native';
+import {Animated, View} from 'react-native';
+import {useOnyx} from 'react-native-onyx';
+// @ts-expect-error This is a workaround to display HelpPane on top of everything,
```

Comment:
I’ve really tried every possible approach 😮‍💨

1. react-native Modal : The modal must be in fullscreen mode; otherwise, the rest of the app becomes unresponsive (users can't click anything outside the side pane)
2. react-native-modal (src/components/Modal/index.tsx): Similar issue as above, but additionally, the side pane was sometimes rendered under other modals in a non-deterministic way
3. Portals: The help pane still appeared under React Native modals
4. react-navigation primitives: Doesn’t work on web
5. Custom overlay with a very high z-index: React Native modals are still rendered on top, causing the side pane to be hidden